### PR TITLE
chore: support debugging devnet process

### PR DESCRIPTION
**Pull request by chokobole**
This commit helps user to debug a process which is run by docker compose. Currently, this only works on Linux. There should be another way for macOS users.

Resolves: #11

### CODE REVIEW

File: specs/meta/devnet.md [hunk 1]
```diff
@@ -51,3 +52,10 @@ The batch submitter uses the account below to submit batches to L1:
 
 - Address: `0xde3829a23df1479438622a08a116e8eb3f620bb5`
 - Private key: `bf7604d9d3a1c7748642b1b7b05c2bd219c9faa91458b370f85e5a40f3b03af7`
+
+## Debugging vscode
```
Comment (Pangssu) [inline specs/meta/devnet.md:56]: VS Code

File: Dockerfile [hunk 2]
```diff
@@ -16,19 +16,55 @@ RUN make build
...
 FROM alpine:3.17 as kanvas-node
 COPY --from=builder /app/bin/kanvas-node /usr/local/bin
+
+RUN addgroup user && \
+    adduser -G user -s /bin/sh -h /home/user -D user && \
+    mkdir /kanvas_log/ && \
+    chown user:user /kanvas_log
+
+USER user
+WORKDIR /home/user/
```
Comment (Pangssu) [inline Dockerfile:26]: I think it's better to separate the stage for adding this.
Is this possible?

```suggestion
FROM alpine:3.17 as runner
RUN addgroup user && \
    adduser -G user -s /bin/sh -h /home/user -D user && \
    mkdir /kanvas_log/ && \
    chown user:user /kanvas_log
USER user
WORKDIR /home/user/

# Node
FROM runner as kanvas-node
COPY --from=builder /app/bin/kanvas-node /usr/local/bin
...

File: Dockerfile [hunk 3]
```diff
@@ -16,19 +16,55 @@ RUN make build
...
 CMD ["kanvas-stateviz"]
 
 # Batcher
 FROM alpine:3.17 as kanvas-batcher
 COPY --from=builder /app/bin/kanvas-batcher /usr/local/bin
+
+RUN addgroup user && \
+    adduser -G user -s /bin/sh -h /home/user -D user && \
+    mkdir /kanvas_log/ && \
+    chown user:user /kanvas_log
```
Comment (Pangssu) [inline Dockerfile:51]: FYI, batcher and validator do not required a `kanvas_log` file.

File: Dockerfile (hunk 2 above)
Comment (Pangssu) [reply Dockerfile:26]: This code will not work and is for reference purposes only.

File: Dockerfile (hunk 3 above)
Comment (chokobole) [reply Dockerfile:51]: Oh thanks i'll remove the lines on `batcher` and `validator`

**Pull request merged**